creator-node/compose/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - delegateOwnerWallet=${delegateOwnerWallet}
       - delegatePrivateKey=${delegatePrivateKey}
       - creatorNodeEndpoint=${creatorNodeEndpoint}
-      - snapbackDevModeEnabled=${snapbackDevModeEnabled}
+      - manualSyncsDisabled=${manualSyncsDisabled}
```

Comment:
we no longer use `snapbackDevModeEnabled` for its original purpose, which was to determine `snapbackJobInterval`. that has been moved to its own independent configvar

We now only use this to determine whether or not to issue manual syncs, so I renamed it to reflect that usage

Reply:
i did?

Reply:
maybe @dmanjunath meant the manifests?

Reply:
yeah i actually don't think we need to change any of the manifests, as all these values already specify prod defaults where needed. right?

Reply:
yup! I believe so

Reply:
sorry i just meant like wherever this gets set to true. i actually couldn't find it

Reply:
well it isn't set to true anywhere, by default it is false - we just use it when devving to flip the mode manually

Reply:
ah okay, so it's set by hand. cool resolving